src/plugins/opencv/nodes/sequence/mean.cpp

```diff
@@ -0,0 +1,94 @@
+#include <possumwood_sdk/node_implementation.h>
+
+#include <boost/noncopyable.hpp>
+
+#include <tbb/parallel_reduce.h>
+#include <tbb/blocked_range.h>
+
+#include "sequence.h"
+
+namespace {
+
+dependency_graph::InAttr<possumwood::opencv::Sequence> a_in;
+dependency_graph::OutAttr<possumwood::opencv::Frame> a_out;
+
+struct Reduce : public boost::noncopyable {
+	Reduce(const possumwood::opencv::Sequence& seq) {
```

Comment:
Codacy found an issue: [Struct 'Reduce' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/martin.prazak/possumwood/pullRequest?prid=5900978)